src/client/datascience/jupyter/jupyterExecution.ts

```diff
@@ -216,20 +216,9 @@ export class JupyterExecutionBase implements IJupyterExecution {
                             traceError('Failed to connect to server', ex);
                             if (ex instanceof JupyterSessionStartError && isLocalConnection && allowUI) {
                                 sendTelemetryEvent(Telemetry.AskUserForNewJupyterKernel);
-
-                                // Keep retrying, until it works or user cancels.
-                                const kernelInterpreter = await this.kernelSelector.askForLocalKernel(
-                                    options.resource,
-                                    connection,
-                                    launchInfo.kernelConnectionMetadata
+                                void this.kernelSelector.askForLocalKernel(
```

Comment:
This is going to throw even though you asked for another kernel. Shouldn't it break?

Reply:
I think its ok for the message to be displayed the startup failed. Else the rest of the code might not work correctly